opencti-platform/opencti-graphql/src/database/file-storage-helper.ts

```diff
@@ -14,6 +14,7 @@ interface FileUploadOpts {
   meta? : any,
   noTriggerImport?: boolean,
   errorOnExisting?: boolean,
+  file_markings?: [],
```

Comment:
By typing it using `[]` you are telling that file_markings is an empty array, need to replace with an array of what, if we don't know the type we can use `unkown[]` or `any[]`

Reply:
ah ok I see

Reply:
Should be string[] I think

Reply:
will be addressed in https://github.com/OpenCTI-Platform/opencti/pull/6868